advanced/am-pattern-1/README.md

```diff
@@ -138,6 +153,7 @@ b. Add the above hosts as entries in `/etc/hosts` file as follows:
 c. Try navigating to `https://<RELEASE_NAME>-am/carbon`, `https://<RELEASE_NAME>-am/publisher` and `https://<RELEASE_NAME>-am/devportal` from your favorite browser.
 
 ## Configuration
+
 The following tables lists the configurable parameters of the chart and their default values.
 
```

Comment:
list?

Reply:
@chamindi-a how about `defines`?

Reply:
but since you are listing 
lets keep the 'list'